index.html

```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <div class="card">
+			<p class="name">Luca Giannotti</p>
+			<p class="contact">
+			  <i class="fab fa-instagram"></i>
+			  <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
```

Comment:
Since you have not provided your Twitter profile, you can remove these lines.

Reply:
Okay, I removed it and sent another commit!